HW1/P5/P5_bfs.py

```diff
@@ -0,0 +1,18 @@
+# Takes in a src node and a destination node, and returns an rdd of
+# every shortest path from src to dest in the graph.
+# Caution: if src is not connected to dest, this runs forever
+def find_all_shortest_paths(src, dest, graph, sc):
+  accum = sc.accumulator(0)
+  num_parts = 64
+  graph = graph.partitionBy(num_parts).cache()
+  distances = sc.parallelize([(src, [src])]).partitionBy(num_parts)
+  while 1:
```

Comment:
I think `while True` may be the more accepted "infinite loop" notation in Python.